JSDemos/Demos/Autocomplete/Overview/Angular/app/app.component.ts

```diff
@@ -24,23 +28,45 @@ export class AppComponent {
 
   positions: string[];
 
-  cities: string[];
-
   states: any;
 
+  clients: CustomStore;
+
   firstName = '';
 
   lastName = '';
 
   position: string;
 
-  city = '';
-
   state = '';
 
+  currentClient = '';
+
   fullInfo = '';
 
-  constructor(service: Service) {
+  constructor(httpClient: HttpClient, service: Service) {
+    function isNotEmpty(value: any): boolean {
+      return value !== undefined && value !== null && value !== '';
+    }
+    this.clients = new CustomStore({
+      key: 'Value',
+      useDefaultSearch: true,
+      load(loadOptions: any) {
+        let params: HttpParams = new HttpParams();
+        [
+          'skip',
+          'take',
+          'filter',
+        ].forEach((i) => {
+          if (i in loadOptions && isNotEmpty(loadOptions[i])) { params = params.set(i, JSON.stringify(loadOptions[i])); }
```

Comment:
I think it's better to use `Object.hasOwn` instead of `in` operator to check only for own properties